### DISCUSSION TITLE
Custom resource is goot to manage via helm chart like kafka , kafkatopic, kafkauser and kafkamirrormaker?

### DISCUSSION THREAD

**Discussion by Veljen** [strimzi/strimzi-kafka-operator#9595] [Q&A]
Hi,

we would like to disable auto  creation of topics.   can we use customized helm chart to manage the CR (kafka, kafkatopic, mirror maker)..  Using helm is it good manage the kafka topics (add/remove).   I would like to know some ideas on that.

**Comment by scholzj** [comment]
Strimzi does not provide a Helm Chart for creating the custom resources. Only to install the Cluster operator. One of the main reasons is that there are simply too many options and variations for something like Kafka custom resource. But yes, you can easily write your own Helm chart for it and in fact many users do it.
